tests/ZendTest/Mvc/Controller/RestfulControllerTest.php

```diff
@@ -137,9 +137,15 @@ public function testDispatchInvokesDeleteMethodWhenNoActionPresentAndDeleteInvok
 
     public function testDispatchInvokesDeleteListMethodWhenNoActionPresentAndDeleteInvokedWithoutIdentifier()
     {
-        $this->request->setMethod('DELETE');
+        $entities = array(
+            array('id' => uniqid(), 'name' => __FUNCTION__),
+            array('id' => uniqid(), 'name' => __FUNCTION__),
+            array('id' => uniqid(), 'name' => __FUNCTION__),
+        );
+        $string = http_build_query($entities);
+        $this->request->setMethod('DELETE')
+                      ->setContent($string);
```

Comment:
While this sets up data to pass to the controller, it does not test that the controller received that data. You need to do something with the data that you can then assert against later.